ui/admin/tests/e2e/tests/target.spec.js

```diff
@@ -59,53 +65,57 @@ test('Verify session created then cancel the session', async ({ page }) => {
     );
     const target = targets.items.filter((obj) => obj.name == targetName)[0];
 
-    connect = exec(
-      'boundary connect' +
-        ' -target-id=' +
-        target.id +
-        ' -exec /usr/bin/ssh --' +
-        ' -l ' +
-        process.env.E2E_SSH_USER +
-        ' -i ' +
-        process.env.E2E_SSH_KEY_PATH +
-        ' -o UserKnownHostsFile=/dev/null' +
-        ' -o StrictHostKeyChecking=no' +
-        ' -o IdentitiesOnly=yes' + // forces the use of the provided key
-        ' -p {{boundary.port}}' +
-        ' {{boundary.ip}}'
-    );
-
+    connect = await connectToTarget(target);
+    await waitForSessionToBeVisible(page, targetName);
     await page
-      .getByRole('navigation', { name: 'Resources' })
-      .getByRole('link', { name: 'Sessions' })
+      .getByRole('cell', { name: targetName })
+      .locator('..')
+      .getByRole('button', { name: 'Cancel' })
       .click();
-    let i = 0;
-    let sessionIsVisible = false;
-    do {
-      i = i + 1;
-      sessionIsVisible = await page
-        .getByRole('cell', { name: targetName })
-        .isVisible();
-      if (sessionIsVisible) {
-        break;
-      }
-      await page.getByRole('button', { name: 'Refresh' }).click();
-      await expect(page.getByRole('button', { name: 'Refresh' })).toBeEnabled();
-    } while (i < 5);
-
-    if (!sessionIsVisible) {
-      throw new Error('Session is not visible');
+  } finally {
+    await deleteOrg(org.id);
+    // End `boundary connect` process
+    if (connect) {
+      connect.kill('SIGTERM');
     }
+  }
+});
+
+test('Verify session created to target with address, then cancel the session', async ({
+  page,
+}) => {
+  await page.goto('/');
+  let org;
+  let connect;
+  try {
+    const orgName = await createNewOrg(page);
+    const projectName = await createNewProject(page);
+    const targetName = await createNewTargetWithAddress(page);
 
+    await authenticateBoundaryCli();
+    const orgs = JSON.parse(execSync('boundary scopes list -format json'));
+    org = orgs.items.filter((obj) => obj.name == orgName)[0];
+    const projects = JSON.parse(
+      execSync('boundary scopes list -format json -scope-id ' + org.id)
+    );
+    const project = projects.items.filter((obj) => obj.name == projectName)[0];
+    const targets = JSON.parse(
+      execSync('boundary targets list -format json -scope-id ' + project.id)
+    );
+    const target = targets.items.filter((obj) => obj.name == targetName)[0];
+
+    connect = await connectToTarget(target);
+    await waitForSessionToBeVisible(page, targetName);
     await page
       .getByRole('cell', { name: targetName })
       .locator('..')
       .getByRole('button', { name: 'Cancel' })
       .click();
   } finally {
+    await deleteOrg(org.id);
     // End `boundary connect` process
     if (connect) {
-      connect.kill('SIGTERM');
+      connect.kill();
```

Comment:
Do we want the `SIGTERM` here?

Reply:
It applies `SIGTERM` under the hood if no argument is provided: The doc:

> If no argument is given, the process will be sent the 'SIGTERM'

We may want to specify it to highlight our intention though